scripts/evaluate.py

```diff
@@ -124,21 +144,31 @@ def import_mobilenetv3_ssdlite(self, target="llvm", dtype="float32"):
         return (mod, params, input_shape, dtype, target)
 
     def import_deeplabv3(self, target="llvm", dtype="float32"):
-        import onnx
-
-        graph_file = os.path.abspath(
+        import tensorflow as tf
+        try:
+            tf_compat_v1 = tf.compat.v1
+        except ImportError:
+            tf_compat_v1 = tf
+        # Tensorflow utility functions
+        import tvm.relay.testing.tf as tf_testing
+
+        model_path = os.path.abspath(
             os.path.dirname(os.path.realpath(__file__))
-            + "/../models/deeplabv3_mnv2_pascal_train_aug/deeplabv3_mnv2.onnx"
-        )
-        model = onnx.load_model(graph_file)
-        input_shape = {"ImageTensor:0": (1, 224, 224, 3)}
-        input_names, shape_dict = get_input_data_shape_dict(
-            model, input_shape["ImageTensor:0"]
+            + "/../models/mace_deeplabv3/deeplab-v3-plus-mobilenet-v2.pb"
```

Comment:
Today I learnt that a single protobuf fite keeps both models - deeplab and mobile net. Just curious, do we know why MACE does this composite model?

Reply:
As far as I understand, Mobilenet is used as an encoder here.
https://www.kaggle.com/mobassir/deeplabv3-with-mobilenetv2